### ISSUE TITLE
many websites have different problems.

### ISSUE THREAD

**Issue by reyis4402** [mozilla-mobile/fenix#16872]
Although I opened the google translation site only once, the history tab shows that I have opened many times at the same time
![Screenshot_20201205-155639_Firefox_Nightly](https://user-images.githubusercontent.com/60405617/101243622-c7b5af80-3701-11eb-9c13-7245e4138587.png)

The github site gives errors when registering or logging in.
![Screenshot_20201205-155749_Galeri](https://user-images.githubusercontent.com/60405617/101243663-0cd9e180-3702-11eb-8290-b62b307d63e4.png)
![Screenshot_20201205-155931_Galeri](https://user-images.githubusercontent.com/60405617/101243674-1f541b00-3702-11eb-8d37-46af4d6a1de3.png)
keeps failing when my facebook account is open
![Screenshot_20201205-160202_Galeri](https://user-images.githubusercontent.com/60405617/101243735-7fe35800-3702-11eb-9020-a1881545a9a0.png)

**Comment by reyis4402** [comment]
these issues exist for the latest firefox nightly release

**Comment by strugee** [comment]
@reyis4402 the Google Translate problem occurs because Google Translate puts a new entry into the tab history every time you type something. You'll see that the same behavior occurs on desktop Firefox and on other browsers.

As for the other problems, please don't post more than one problem in the same GitHub issue - it makes it really hard for developers to keep track of the different problems. Instead, open a new GitHub issue for each problem. It's okay (and preferred) to open multiple issues in a row to avoid putting everything in the same report like you've done here.

I'm sure the fine folks at Mozilla appreciate your willingness to help out!

**Comment by sflorean** [comment]
@reyis4402 please file new issue for the bugs you find. I will close this because it is hard to follow even if we edit. Thanks for understanding and report!
